elliott/elliottlib/cli/snapshot_cli.py

```
@@ -88,6 +132,115 @@ async def _get(db: KonfluxDb, nvrs: list[str]) -> list[KonfluxRecord]:
+
+    Args:
+        runtime: The Elliott runtime instance containing group information
+        provided_namespace: Explicitly provided namespace (takes precedence)
+
+    Returns:
+        Resolved namespace (guaranteed to return a valid namespace)
+    """
+    if provided_namespace:
+        return provided_namespace
+
+    matched_prefix = None
```

Ruff (0.14.2)

Local variable `matched_prefix` is assigned to but never used

Remove assignment to unused variable `matched_prefix`

(F841)